torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -291,16 +291,32 @@ Tensor mul_tensor_backward(Tensor grad, Tensor other, ScalarType self_st) {
   return handle_r_to_c(self_st, out);
 }
 
-Tensor div_tensor_self_backward(Tensor grad, Tensor other, ScalarType self_st) {
+Tensor div_tensor_self_backward(Tensor grad, Tensor other, ScalarType self_st, c10::string_view rounding_mode) {
+  if (rounding_mode != "true") {
+    return at::zeros_like(grad, grad.options().dtype(self_st));
```

Comment:
@albanD to sanity-check the gradients. The floor and trunc operations do return zeros, although floor_divide simply has no gradient support today.

Reply:
Our story about gradients for piecewise constant functions is not the best. But as of today, returning zero gradients is the thing to do I think. In particular, it matches the behavior we already have for functions like `sign` and I would argue that consistency is the most important thing here.